admin/kernel-masterdata-service/src/main/java/io/mosip/kernel/masterdata/service/impl/DeviceServiceImpl.java

```diff
@@ -266,20 +263,20 @@ public IdResponseDto deleteDevice(String id) {
 		try {
 			foundDeviceList = deviceRepository.findByIdAndIsDeletedFalseOrIsDeletedIsNull(id);
 
-			if (foundDeviceList!=null && !foundDeviceList.isEmpty()) {
+			if (foundDeviceList != null && !foundDeviceList.isEmpty()) {
```

Comment:
reverse this condition so its more readable. so you need not have else.

Reply:
taken care